modules/swagger-codegen/src/main/resources/csharp/model.mustache

```diff
@@ -13,7 +13,7 @@ namespace {{package}} {
   /// {{description}}
   /// </summary>
   [DataContract]
-  public class {{classname}} {
+  public class {{classname}}{{#parent}} : {{{parent}}}{{/parent}} {
```

Comment:
Since this PR is for PHP, I would suggest you to roll back the change for this file. 

I (or the author of https://github.com/swagger-api/swagger-codegen/commit/5f1df9e093eaf7b584c98879b4e05dcf1ac8c58f) will submit another one for develop_2.0